### PR TITLE
(#1046) - Implement `Bytes#sshift`

### PR DESCRIPTION
**Pull request by andreoss**
Per #1046

### CODE REVIEW

File: eo-runtime/src/main/java/org/eolang/BytesOf.java [hunk 1]
```diff
@@ -208,4 +236,34 @@ public boolean equals(final Object other) {
     public int hashCode() {
         return Arrays.hashCode(this.data);
     }
+
+    /**
+     * Count leading zero bits.
+     * @param num Byte.
+     * @return Number between 0 and 8.
+     */
+    static byte numberOfLeadingZeros(final byte num) {
```
Comment (mximp) [inline eo-runtime/src/main/java/org/eolang/BytesOf.java:245]: @andreoss can this be private? There is no reason to have this package-private other than tests. Exposing members solely to test them is a design flaw.

Comment (andreoss) [reply eo-runtime/src/main/java/org/eolang/BytesOf.java]: @mximp Done

File: eo-runtime/src/main/java/org/eolang/BytesOf.java [hunk 2]
```diff
@@ -208,4 +236,34 @@ public boolean equals(final Object other) {
...
     }
+
+    /**
+     * Count leading zero bits.
+     * @param num Byte.
+     * @return Number between 0 and 8.
+     */
+    private static byte numberOfLeadingZeros(final byte num) {
+        final byte result;
+        if (num <= 0) {
```
Comment (mximp) [inline eo-runtime/src/main/java/org/eolang/BytesOf.java:247]: @andreoss This conditions can be combined:
```
if (num < 0) {
} else if (num == 0) {
} else {
}
```

**Comment by yegor256** [comment]
@andreoss @mximp what's with this?

**Review by mximp — APPROVED** [review]

**Comment by andreoss** [comment]
@yegor256 this PR can be merged

**Comment by andreoss** [comment]
@yegor256 Please, merge

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**